frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -832,8 +832,8 @@ end
 
 function ReaderDictionary:showDownload(downloadable_dicts)
     local kv_pairs = {}
-    table.insert(kv_pairs, {_("Tap dictionary name to download"), ""})
-    table.insert(kv_pairs, "----------------------------")
+    -- table.insert(kv_pairs, {_("Tap dictionary name to download"), ""})
```

Comment:
Could you also remove the commented lines? ^_^

Reply:
Sure